backend/main.py

```
@@ -90,10 +90,21 @@
+    'yantrax_agent_latency_seconds_count': 0,
+    'yantrax_agent_latency_seconds_sum': 0.0
+}
+
 def handle_errors(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         error_counts['total_requests'] += 1
+        try:
+            metrics_registry['yantrax_requests_total'] += 1
+        except Exception:
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)